src/daft-core/src/utils/sketch.rs

```diff
@@ -0,0 +1,51 @@
+use common_error::DaftResult;
+use sketches_ddsketch::{Config, DDSketch};
+
+pub struct Sketch {
```

Comment:
You can also define pure-wrapper structs like so: `pub struct Sketch(DDSketch)`. You can then access the inner type using `self.0`, but all clients of `Sketch` can only access the `pub` methods.